Added description of day of week method (#3836)
Codecademy/docs: content/kotlin/concepts/datetime/terms/DayOfWeek/DayOfWeek.md

---

> Title: 'DayOfWeek'

Reviewer: ```suggestion
Title: '.DayOfWeek'
```

> Description: 'The DayOfWeek method will return the '

Reviewer: ```suggestion
Description: 'Useful tool for working with weekdays in the Kotlin code, making it easier to perform various operations and checks related to days of the week.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Method'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

> The **DayOfWeek** method is used to return the DayOfWeek instance for the week day number, starting with 1 as Monday. 

Reviewer: ```suggestion
The **`DayOfWeek`** method is used to return the `DayOfWeek` instance for the weekday number, starting with `1` as `Monday`. 
```

> ## Syntax

Reviewer: ```suggestion

## Syntax

```
We have style guide to leave a white space after heading sub-heading and if a paragraph ends.

'''pseudo
fun DayOfWeek(isoDayNumber: Int): DayOfWeek
'''

> The integer value input into the method must correspond to one of the seven days in the DayOfWeek enumeration class. The numbering follows the ISO-8601 week day number, where Monday is 1 and Sunday is 7. 
> 
> The enumeration class enteries have two properties: Name, which is represented by a string, and the ordinal, which is represented as an integer. 

Reviewer: ```suggestion

- The `integer` value input into the method must correspond to one of the seven days in the `DayOfWeek` enumeration class. The numbering follows the ISO-8601 weekday number, where `Monday` is `1` and `Sunday` is `7`. 
- The enumeration class entries have two properties: `Name`, which is represented by a `string`, and the `Ordinal`, which is represented as an `integer`. 
```

> ## Example

Reviewer: ```suggestion
## Example

```
Again a white space is required here

'''kolin

> println(DayOfWeek(3))

Reviewer: ```suggestion
import java.time.DayOfWeek

fun main() {
    val day = DayOfWeek.of(3)
    println(day)
}
```
Made some changes in this code snippet so it can work if someone tries to compile it.

'''

> This will output a string "Wednesday". The valueof method may be used to determine the integer value of a string in the DayOfWeek enum. Do note that the value must match exactly.

Reviewer: This is not the right way to add output. The right way is to use ```shell like you created the above code snippet example. Instead of the lang tag we use shell for outputs. Please refactor this.

---
Title: '.DayOfWeek'
Description: 'Useful tool for working with weekdays in the Kotlin code, making it easier to perform various operations and checks related to days of the week.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Method'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

The **`DayOfWeek`** method is used to return the `DayOfWeek` instance for the weekday number, starting with `1` as `Monday`. 

## Syntax

'''pseudo
fun DayOfWeek(isoDayNumber: Int): DayOfWeek
'''

- The `integer` value input into the method must correspond to one of the seven days in the `DayOfWeek` enumeration class. The numbering follows the ISO-8601 weekday number, where `Monday` is `1` and `Sunday` is `7`. 

> - The enumeration class entries have two properties: `Name`, which is represented by a `string`, and the `Ordinal`, which is represented as an `integer`. 

Reviewer: ```suggestion
- The enumeration class entries have two properties: `Name`, which is represented by a `string`, and the `Ordinal`, which is represented as an `integer`. 

```

## Example

> '''kolin

Reviewer: Add a basic intro above this code snippet
